.env.example

```diff
@@ -1,14 +0,0 @@
-PORTAL_RUNNER_USERNAME=
-PORTAL_RUNNER_PASSWORD=
```

Comment:
add this file back

Reply:
add these 2 variables as well:
PORTAL_RUNNER_MONGO_DATABASE_ACCOUNT=
PORTAL_RUNNER_MONGO_DATABASE_ACCOUNT_KEY=

Reply:
why are these removed? add them back